zebra-consensus/Cargo.toml

```diff
@@ -5,21 +5,27 @@ authors = ["Zcash Foundation <zebra@zfnd.org>"]
 license = "MIT OR Apache-2.0"
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-zebra-chain = { path = "../zebra-chain" }
-zebra-state = { path = "../zebra-state" }
 chrono = "0.4.12"
+futures = "0.3.5"
+futures-core = "0.3.5"
 futures-util = "0.3.5"
-tower = "0.3.1"
+rand = "0.7"
+redjubjub = { git = "https://github.com/ZcashFoundation/redjubjub.git", branch = "main"}
```

Comment:
we should make a tracking issue to remove this once there's a new rjj release (no need to block on it here)

Reply:
https://github.com/ZcashFoundation/zebra/issues/606